frontera/settings/default_settings.py

```diff
@@ -50,6 +51,7 @@
 }
 SQLALCHEMYBACKEND_REVISIT_INTERVAL = timedelta(days=1)
 STATE_CACHE_SIZE = 1000000
+STATE_CACHE_SIZE_LIMIT = 0
```

Comment:
I think we should be using a single option for all backends. All these options adjust the same value but in different backends.
``STATE_CACHE_SIZE``
``HBASE_STATE_CACHE_SIZE_LIMIT``
``STATE_CACHE_SIZE_LIMIT``

In SQLA the STATE_CACHE_SIZE should be used instead of _LIMIT.